commitlint.config.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2019 American Express Travel Related Services Company, Inc.
```

Comment:
The year should be 2020 now right?>

Reply:
This still has not been fixed.

Reply:
Fixed in a1dba5e. Missed it during the rebase with master.